build-pipeline.yml

```diff
@@ -44,10 +45,6 @@ parameters:
   - name: has_ui_tests
     type: boolean
     default: false
-  # Tells the build pipeline that there are python 2.7 tests to execute
-  - name: has_python_27_tests
```

Comment:
This parameter seems to still be used in shotgunsoftware/tk-framework-desktopserver

Reply:
[You're right](https://github.com/shotgunsoftware/tk-framework-desktopserver/blob/master/azure-pipelines.yml#L34).

However, it is passing `false`. So there won't be any issue here. We still need to clean up that parameter on desktopserver anyway.